New package: SmoQyDEAC v0.1.5

JuliaRegistrator opened the pull request:
- Registering package: SmoQyDEAC
- Repository: https://github.com/SmoQySuite/SmoQyDEAC.jl
- Created by: @sandimas
- Version: v0.1.5
- Commit: 3718bdcdbd2ede64c4fb80cdbd003d8af2205ff4
- Git reference: HEAD

sandimas commented:
Resubmitting with (hopefully) issues fixed for auto merge

sandimas commented:
SmoQyDEAC is part of the same suite of code as SmoQyDQMC. 

DEAC = Differential Evolution for Analytic Continuation
DQMC = Determinantal Quantum Monte Carlo

goerz commented:
[noblock] @sandimas You have to edit your comments to contain `[noblock]`

sandimas commented:
[noblock] thanks!

JuliaHubRegistrator committed:
New package: SmoQyDEAC v0.1.5

UUID: 9c442b74-637f-4296-b43a-959aa316565a
Repo: https://github.com/SmoQySuite/SmoQyDEAC.jl.git
Tree: 01224a52f7b8eae7ce938af3c21d296dd36a0c7a

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09
